eZ/Publish/API/Repository/Tests/BaseTest.php

```diff
@@ -622,4 +623,41 @@ protected function assertValidationErrorOccurs(
 
         self::assertThat($exception, $constraint);
     }
+
+    /**
+     * Create 'folder' Content.
+     *
+     * @param array $names Folder names in the form of <code>['&lt;language_code&gt;' => '&lt;name&gt;']</code>
+     * @param int $parentLocationId
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Location
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    protected function createFolder(array $names, int $parentLocationId): Location
+    {
+        $repository = $this->getRepository(false);
+        $contentService = $repository->getContentService();
+        $contentTypeService = $repository->getContentTypeService();
+        $locationService = $repository->getLocationService();
+
+        $mainLanguageCode = array_keys($names)[0];
```

Comment:
potentially $names could be an empty array

Reply:
That would be bad, added extra check via 0dc27d3.